odoo.conf

```diff
@@ -1,7 +1,7 @@
 [options]
 admin_passwd = admin
 debug=True
-dev=True
+dev=all
```

Comment:
This modification should not stand in this PR. I don't use this configuration file, feel free to fix it in another PR or commit. :)

Reply:
I've fixed it. :)